policyengine_us/parameters/gov/states/ny/tax/income/deductions/itemized/college_tuition/applicable_percentage.yaml

```diff
@@ -0,0 +1,13 @@
+description: New York provides an itemized deduction for this fraction of allowable college tuition expenses.
+values:
+  2001-01-01: 0.25
+  2002-01-01: 0.5
+  2003-01-01: 0.75
+  2004-01-01: 1
+metadata:
+  unit: /1
+  period: year
+  label: New York college tuition deduction applicable percentage
+  reference:
+    - title: Section 615 Credits against tax (d)(4)
```

Comment:
Are these rates mentioned in the tax forms?

Reply:
No because at 100% it'd be superfluous